### PR TITLE
feat: updated contribution guidelines

### PR DESCRIPTION
**Pull request by vedant-jain03**
Contributed by: @CircuitVerse/issues-team 
Guided by: @Shivansh2407 @tachyons @ayan-biswas0412

### CODE REVIEW

File: contributing_guidelines.md [hunk 1]
```diff
@@ -1,414 +1,89 @@
...
+- [no activity](https://github.com/CircuitVerse/Interactive-Book/labels/no%20activity) labeled PR are meant to have no activity in the PR from since a while.
+- [blocked](https://github.com/CircuitVerse/Interactive-Book/labels/blocked) labeled PR are meant to not to go ahead for review.
+- [do not merge](https://github.com/CircuitVerse/Interactive-Book/labels/do%20not%20merge) labeled PR are meant to not to merge the PR right now(may be later).
+- [awaiting-approval](https://github.com/CircuitVerse/CircuitVerse/labels/awaiting-approval) labeled PR are meant to be waiting for other communtiy members.
 
-To work locally, you will need to setup a Jekyll development environment. It is posible to setup Jekyll natively for your OS or run it in a docker container.
+## Community
+Discussions about Interactive Book issues and features take place on the repository's [Discussions](https://github.com/CircuitVerse/CircuitVerse/discussions) sections. Anybody is welcome to join these conversations. See the [README](README.md) for more information on communication channels.
```
Comment (ayan-biswas0412) [inline contributing_guidelines.md:80]: I think slack should be here instead of discussion

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
Please resolve the comments by Github Actions.

File: contributing_guidelines.md (hunk 1 above)
Comment (vedant-jain03) [reply contributing_guidelines.md]: Done

**Review by ayan-biswas0412 — APPROVED** [review]
Good work @vedant-jain03 🎉

**Review by Kartikay304 — APPROVED** [review]

**Review by tachyons — CHANGES_REQUESTED** [review]
Interactive book requires different contribution guidelines as it is primarily about digital circuit learning, so it is important to keep the existing guidelines

**Comment by vedant-jain03** [comment]
> Interactive book requires different contribution guidelines as it is primarily about digital circuit learning, so it is important to keep the existing guidelines

Sir should we add some specific section into existing one such as:
- Issue label
- Pull request reviews
- PR Labels
- Community

**Comment by tachyons** [comment]
We don't need that for the time being

**Pull request closed without merging**